nanshe_ipython.ipynb

```diff
@@ -378,6 +382,7 @@
         "med_filt_size = 3\n",
         "norm_filt_sigma = 100\n",
         "\n",
+        "block_frames = 100\n",
```

Comment:
In reality, this default may be totally wrong, but having the option to set it is definitely the way to go.